hackathon_site/review/jinja2/review/emails/accepted_email_body.html

```diff
@@ -61,7 +61,7 @@
                                         <br><br>
                                         Read our <a href="{{ participant_package_link }}">participant package</a> for all the info regarding the event, and join our <a href="{{ chat_room_link }}">{{ chat_room_name }}</a>. Stay tuned for more updates regarding detailed event logistics, and we hope to see you soon!
                                         <br><br>
-                                        As well, our <a href="https://hardware.makeuoft.ca">Hardware Signout Site</a> will be open for viewing starting February 9th at 11:00 AM. Please remember to recreate your team on the site once it's open for viewing. Starting February 12th at 11:59 AM, you will be able to place orders and borrow hardware items from our inventory. Get ready to explore our offerings and prepare your team in advance!
+                                        As well, our <a href="https://hardware.makeuoft.ca">Hardware Signout Site</a> will be open for viewing starting February 11th at 11:59 PM. Please remember to recreate your team on the site once it's open for viewing. Starting February 11th at 11:59 PM, you will be able to place orders and borrow hardware items from our inventory. Get ready to explore our offerings and prepare your team in advance!
```

Comment:
I'm guessing viewing and when they can start ordering is the same date, does this align when acceptances go live?